modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -0,0 +1,515 @@
+
+```yml
+# Examples
+
+check: this.type == "end_of_transaction"
+```
+
+=== `batching.processors`
+
+For aggregating and archiving message batches, you can add a list of xref:components:processors/about.adoc[processors] to apply to a batch as it is flushed. All resulting messages are flushed as a single batch even when you configure processors to split the batch into smaller batches.
+
+*Type*: `array`
+
+```yml
+# Examples
+
+processors:
+  - archive:
+      format: concatenate
+
+processors:
+  - archive:
+      format: lines
+
+processors:
+  - archive:
+      format: json_array
+```
+
+=== `max_in_flight`
+
+The maximum number of messages to have in flight at a given time. Increase this number to improve throughput until performance plateaus.
+
+*Type*: `int`
+
+*Default*: `64`
+
+=== `channel_prefix`
+
+The prefix to use when creating a channel name for connecting to a Snowflake table. Without a value for `channel_prefix`, this output creates a channel name based on a table's fully qualified name, which results in a single stream per table. 
+
+Adding a `channel_prefix` also avoids the creation of duplicate channel names, which can cause errors and prevent multiple instances of Redpanda Connect from writing at the same time. 
+
+The maximum number of channels open at any time is determined by the value in the `max_in_flight` field. 
+
+NOTE: There is a hard limit of 10,000 streams per table. If you need to use more than 10,000 streams, contact Snowflake support.
+
+*Type*: `string`
+
+== Example pipelines
+
+The following examples show you how to ingest, process, and write data to Snowflake from:
+
+* A Redpanda cluster
+* A REST API that posts JSON payloads to a HTTP server
+
+[tabs]
+======
+Ingesting data from Redpanda::
++
+--
+Ingest data from Redpanda using consumer groups, decode the schema using the schema registry, then write the corresponding data into Snowflake.
+
+NOTE: To create multiple Redpanda Connect streams to write to each output table, you need a unique channel prefix per stream. In this example, the `channel_prefix` field constructs a unique prefix for each stream using the host name.
+
+```yaml
+input:
+  kafka_franz:
+    seed_brokers: ["redpanda.example.com:9092"]
+    topics: ["my_topic_going_to_snow"]
+    consumer_group: "redpanda_connect_to_snowflake"
+    tls: {enabled: true}
+    sasl:
+      - mechanism: SCRAM-SHA-256
+        username: MY_USER_NAME
+        password: "${TODO}"
+pipeline:
+  processors:
+    - schema_registry_decode:
+        url: "redpanda.example.com:8081"
+        basic_auth:
+          enabled: true
+          username: MY_USER_NAME
+          password: "${TODO}"
+output:
+  snowflake_streaming:
+    account: "MYSNOW-ACCOUNT"
+    user: MYUSER
+    role: ACCOUNTADMIN
+    database: "MYDATABASE"
+    schema: "PUBLIC"
+    table: "MYTABLE"
+    private_key_file: "my/private/key.p8"
+    channel_prefix: "snowflake-channel-for-${HOST}"
+```
+--
+HTTP sidecar to push data to Snowflake::
```

Comment:
Can we replace `sidecar` with `server`? Thanks!

Reply:
Updated.